src/views/hoc/hoc.scss

```diff
@@ -8,123 +8,117 @@ $base-bg: $ui-white;
 }
 
 .hoc {
-    .title-banner {
-        &.wbb-bg {
-            background-image: url("/images/hoc2015/hide-bg.jpg");
+    .img,
+    .button {
+        &.card-info {
+            width: calc(100% - 20px);
         }
+    }
 
-        &.dance-bg {
-            background-image: url("/images/hoc2015/dance-bg.jpg");
+    .img {
+        &.card-info {
+            margin: 10px 10px 5px 10px;
+            border-radius: 5px;
         }
 
-        &.name-bg {
-            background-image: url("/images/hoc2015/name-bg.jpg");
+        &.mod-border {
+            border-radius: 5px;
+            width: 100%;
         }
 
-        section {
-            border: 0;
-            padding: 10px 0;
-            max-width: $desktop;
+        &.mod-resource,
+        &.mod-studio {
+            margin-right: 15px;
+        }
 
-            .card-deck {
-                padding: 0 20px;
-            }
+        &.mod-studio {
+            float: left;
+        }
 
-            //6 columns
-            @media only screen and (min-width: $mobile) and (max-width: $tablet - 1) {
-                max-width: $mobile;
-            }
+        &.mod-logo {
+            margin: 20px;
+            max-width: 150px;
+            max-height: 55px;
+            vertical-align: middle;
         }
     }
 
-    .flex-row {
-        .card {
-            margin: 10px;
-            border-radius: 7px;
-            background-color: $active-gray;
-            padding: 2px;
-
-            width: 30%;
-            min-width: 200px;
-            max-width: 230px;
-
-            .card-info {
-                border-radius: 5px;
-                background-color: $base-bg;
-                width: 100%;
-                height: 100%;
-
-
-                button,
-                img {
-                    width: calc(100% - 20px);
-                }
-
-                img {
-                    margin: 10px 10px 5px 10px;
-                    border-radius: 5px;
-                }
-
-                button {
-                    margin: 0 10px 10px 10px;
-                }
-            }
-
-            //8 columns
-            @media only screen and (min-width: $tablet) and (max-width: $desktop - 1) {
-                margin: 2px;
-                min-width: 175px;
-
-                button {
-                    font-size: .7em;
-                }
-            }
-
+    .button {
+        &.card-info {
+            margin: 0 10px 10px 10px;
         }
     }
 
-    section {
+    .section {
```

Comment:
`.section` seems pretty broadly named for something defined within `.hoc`. Maybe this should be split out into a separate `Section` component?